identity/src/main/java/com/android/identity/credentialtype/CredentialTypeRepository.kt

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2023 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.android.identity.credentialtype
+
+/**
+ * A class that contains the metadata of Credential Types
+ */
+object CredentialTypeRepository {
+    private val credentialTypes: MutableList<CredentialType> = mutableListOf()
+
+    /**
+     * Add the Credential Types that are known in this project
+     */
+    init {
+        addCredentialType(DrivingLicense.getCredentialType())
+        addCredentialType(VehicleRegistration.getCredentialType())
+        addCredentialType(VaccinationDocument.getCredentialType())
+        addCredentialType(EUPersonalID.getCredentialType())
+    }
```

Comment:
I think it would be better to not add the built-in types by default. For example, a wallet app will likely add its own types with translations matching the current locale... and if codesize becomes a concern we might even consider putting the concrete types in a separate library so wallet apps that don't need them can easily opt out.

Reply:
Ok, I moved this to `HolderApp.kt` and will add it also to the `VerifierApp.kt` when implementing the usage of the document type repository in the reader app.